src/applications/vaos/containers/ConfirmationPage.jsx

```diff
@@ -91,12 +94,15 @@ ConfirmationPage.propTypes = {
 };
 
 function mapStateToProps(state) {
+  const data = getFormData(state);
+
   return {
-    data: getFormData(state),
+    data,
     facilityDetails: getChosenFacilityDetails(state),
     clinic: getChosenClinicInfo(state),
     flowType: getFlowType(state),
     appointmentLength: getAppointmentLength(state),
+    systemId: getSystemFromParent(state, data?.vaParent),
```

Comment:
Could use `getSystemFromChosenFacility` instead, will save you passing the second param.

Reply:
Nice, will do!